warehouse/macros/gtfs_rt_stg_outcomes.sql

```diff
@@ -25,10 +25,10 @@ stg_gtfs_rt__agg_outcomes AS (
         base64_url,
         {% if step == 'parse' %}
         header,
-        JSON_VALUE(`extract`.response_headers, '$."Last-Modified"') AS last_modified_string,
-        CASE WHEN JSON_VALUE(`extract`.response_headers, '$."Last-Modified"') like '%GMT' THEN PARSE_TIMESTAMP("%a, %d %b %Y %H:%M:%S GMT", JSON_VALUE(`extract`.response_headers, '$."Last-Modified"'))
+        JSON_VALUE(`extract`.response_headers, '$."last_modified"') AS last_modified_string,
+        CASE WHEN JSON_VALUE(`extract`.response_headers, '$."last_modified"') like '%GMT' THEN PARSE_TIMESTAMP("%a, %d %b %Y %H:%M:%S GMT", JSON_VALUE(`extract`.response_headers, '$."last_modified"'))
              -- 1/28/2020 6:29:01 PM
-             WHEN JSON_VALUE(`extract`.response_headers, '$."Last-Modified"') like '%PM' THEN PARSE_TIMESTAMP("%m/%d/%Y %I:%M:%S %p", JSON_VALUE(`extract`.response_headers, '$."Last-Modified"'))
+             WHEN JSON_VALUE(`extract`.response_headers, '$."last_modified"') like '%PM' THEN PARSE_TIMESTAMP("%m/%d/%Y %I:%M:%S %p", JSON_VALUE(`extract`.response_headers, '$."last_modified"'))
```

Comment:
thanks for changing this; just out of curiosity is this a JSON lookup thing in BigQuery? the actual HTTP header really is `Last-Modified`

Reply:
haha I have no idea!

Reply:
it's odd because it works differently for the schedule equivalent of this field: https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.int_gtfs_schedule__joined_feed_outcomes